File: 231213/최고의 13위치/best-place-of-13.py
N = int(input())
arr = [list(map(int, input().split())) for _ in range(N)]

maxCount = 0

for i in range(N):
    for j in range(N - 2):
        tempCount = 0

        box1 = arr[i][j]
        box2 = arr[i][j + 1]
        box3 = arr[i][j + 2]

        if box1 == 1:
            tempCount += 1
        
        if box2 == 1:
            tempCount += 1
        
        if box3 == 1:
            tempCount += 1

        maxCount = max(maxCount, tempCount)

print(maxCount)

File: 231213/최고의 13위치/best-place-of-13.js
const fs = require('fs')
const inputAll = () => fs.readFileSync('/dev/stdin').toString().split('\n');
const print = (...arr) => console.log(...arr)

let [N, ...arr] = inputAll()

arr = arr.map(line => line.split(' ').map(elm => Number(elm)));

let maxCount = 0;

for(let i = 0; i < N; i++){
    for(let j = 0; j < N -2; j++){
        const tempCount = arr[i][j] + arr[i][j + 1] + arr[i][j + 2];
        maxCount = Math.max(tempCount, maxCount);
    }
}

print(maxCount);